workinggroups/ci.md

```diff
@@ -0,0 +1,34 @@
+# CNCF CI WG Proposal
+
+## TOC Sponsor
+
+Camille Fournier
+
+## Objective
+
+Explore the intersection of cloud native and CI technology. Discuss options for taking some of the cluster resources and dedicating them to supporting an open source CI system that can be used by CNCF projects for their CI needs.
+
+## Goals and Expected Outcomes
+
+* We believe that it would be good for us to provide CI services to projects who need or want to use them
+* We need to understanding what, if any, SLA we can promise projects for this system
+* We need to scope what features this system will provide; there is some concern around trying to promise testing the full cross-product of integration with all of the different CNCF projects
+* We want to come away with a recommendation for staffing to support building out this initiative given project needs and desired SLA
+
+## Non Goals
+
+* Run CI for CNCF projects
```

Comment:
Does this non-goal conflict with the first goal of:

> We believe that it would be good for us to provide CI services to projects who need or want to use them